contents/docs/experiments/no-code-web-experiments.md

```diff
@@ -6,126 +6,77 @@ showTitle: true
-
-3. Apply the desired transformations to implement the changes.
-
-The toolbar will guide you through the process of setting up the variant changes visually on your site, making it easier to customize without coding.
-
-![Toolbar launched](https://res.cloudinary.com/dmukukwp6/image/upload/create_web_exp_toolbar_launched_3b23c18d3f.png)
-
-
-### How to edit an experiment variant
-
-Experiment variants enable you to implement and test changes to specific elements on your webpage. For example, you can test different call-to-action (CTA) button sizes and evaluate their effectiveness compared to the baseline (control group).
-
+The toolbar enables you to select elements and apply transformations for each variant in your experiment. Once you launch it:
 
-#### 1. View and edit variants
+1. Click the **vial** tab on the toolbar.
 
-On the experiment detail page, each variant is listed with its rollout percentage. For example:
-  - **control**: `rollout: 33%`
-  - **larger_cta**: `rollout: 33%`
-  - **smaller_cta**: `rollout: 34%`
+2. Select the variant you want to modify the site for. In our case, we select the **test** variant.
 
-You can click on a variant to view or edit it:
-  - Review the variant name (e.g., `larger_cta`).
-  - If no modifications are defined yet, a message such as _"This experiment variant doesn't modify any elements"_ will appear.
-  - Use the options to add or remove elements.
+3. Click **Add element** and select the element you want to modify. We select the main signup button on our page.
 
-#### 2. Add or modify elements
+4. Use the toolbar to modify the element's text, CSS, or HTML. We'll change the button's text to say "Sign up for free".
 
-To make changes to a variant:
-  - Click **Add element** to include specific webpage elements (e.g., buttons, headers, images) that you want to modify.
-  - Define the modifications (e.g., increase button size, change text color).
+5. Once done modifying elements for your variants, click **Save experiment**.
 
-#### 3. Add new variants
+![Implementing web experiments with the toolbar](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_16_at_13_56_01_2x_aff05d4224.png)
 
-To create a new variant:
-  - Click the **Add variant** button at the bottom of the editor.
-  - Enter a name for the new variant.
-  - Add and configure elements for the new variant.
+### Further editing your experiment
 
-#### 4. Save changes
+The toolbar also enables further editing of your experiment. For example, you can:
 
-Once all modifications are complete:
-  - Click the **Save experiment** button to apply the changes.
+- Add multiple element changes to a variant by clicking **+ Add element**.
+- Remove an element from a variant by clicking the trash bin icon.
+- Add variants by clicking **+ Add variant**.
+- Change all elements, headers, buttons, and/or images by clicking the **Change element** dropdown.
```

Comment:
@jurajmajerik I think this is a confusing feature. We might want to review usage / drop it entirely.